client/src/partials/price_list/modal.html

```diff
@@ -0,0 +1,38 @@
+<form  name="ModalForm" ng-submit="ModalCtrl.submit(ModalForm.$invalid)" novalidate>
+  <div class="modal-header">
+    <h4>{{ 'PRICE_LIST.PRICE_LIST_ITEMS' | translate }}</h4>
+  </div>
+
+  <div class="modal-body">
+
+    <div class="form-group" ng-class="{ 'has-error' : ModalForm.$submitted && ModalForm.label.$invalid }">
+      <label class="control-label">{{ 'COLUMNS.DESIGNATION' | translate }}</label>
+      <input type="text" class="form-control" name="label" ng-model="ModalCtrl.data.label" required>
+    </div>
+
+    <div class="form-group" ng-class="{ 'has-error' : ModalForm.$submitted && ModalForm.value.$invalid }">
+      <label class="control-label">{{ 'COLUMNS.VALUE' | translate }}</label>
+      <input type="number" class="form-control" name="value" ng-model="ModalCtrl.data.value" required>
+    </div>
+
+    <div class="form-group" ng-class="{ 'has-error' : ModalForm.$submitted && ModalForm.is_percentage.$invalid }">
+      <label class="control-label">{{ 'COLUMNS.IS_PERCENT' | translate }}</label>
+      <input type="checkbox" class="form-control" name="is_percentage" id="is_percentage" ng-true-value="1" ng-false-value="0" ng-model="ModalCtrl.data.is_percentage">
```

Comment:
The proper way to do checkboxes in bootstrap is shown [here](http://getbootstrap.com/css/#forms-example).  The code should look something like:
```html
  <div class="checkbox">
    <label>
      <input type="checkbox"> Check me out
    </label>
  </div>
```